### PR TITLE
Olwa update

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

**Comment by oliver-walker-hms** [comment]
I had a minute so i went through this again

File: README.md [hunk 1]
```diff
@@ -281,6 +287,31 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the sc-flexy-logger-lib to ensure all logging follows HMS standards for general connectors.
```
Comment (alexjhawk) [inline README.md:291]:
```suggestion
The OSIsoft connector makes use of the HMS Solution Center logging library (sc-flexy-logger-lib) to ensure all logging follows HMS standards for general connectors.
```

File: README.md [hunk 2]
```diff
@@ -281,6 +287,31 @@ The Client ID and the Client Secret will be needed for the basic script you will
...
+More information on each logging level can be found here: [Logging Library](https://github.com/hms-networks/sc-flexy-logger-lib)
+
+To add a different logging level in your configuration file specify the logging level under the AppConfig object.
```
Comment (alexjhawk) [inline README.md:294]:
```suggestion
To add a different logging level in your configuration file, specify the logging level under the AppConfig object.
```

File: README.md [hunk 3]
```diff
@@ -281,6 +287,31 @@ The Client ID and the Client Secret will be needed for the basic script you will
...
+This is an optional field in the configuration file and is not required by the application to properly run.
```
Comment (alexjhawk) [inline README.md:295]:
```suggestion
LoggingLevel is an optional field in the configuration file and is not required by the application to properly run.
```
What is "this"?

File: README.md [hunk 4]
```diff
@@ -281,6 +287,30 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib) to ensure all logging follows HMS standards for general connectors.
+
+To add a different logging level in your configuration file, specify the logging level under the AppConfig object.
+LoggingLevel is an optional field in the configuration file and is not required by the application to properly run.
+
+Below is an example of the configuration file set to log out general information.
```
Comment (TomKimsey) [inline README.md:296]:
```suggestion
Below is an example of the configuration file set to log general application state information. See [Log Level](https://github.com/hms-networks/sc-flexy-logger-lib#log-level) for more information.
```

File: README.md [hunk 5]
```diff
@@ -16,21 +16,27 @@ This application is supported by HMS' North American offices.
...
+4. [Configuration File](#Configuration-File)
+5. [Customizing The Application](#Customizing-the-application)
+6. [Tag Configuration](#Tag-Configuration)
+7. [OMF Support](#OMF-Support)
+8. [OSIsoft Cloud Service](#OSIsoft-Cloud-Service)
+9. [JVM Logger](#JVM-Logger)
+10. [Troubleshooting Common Issues](#Troubleshooting-common-issues)
+
+## General Information
+The OSIsoft connector makes use of the Flexy's historical logging functionality to send historical data points from a Flexy to OSIsoft PIWebApi. Due to the Flexy using its historical logging feature, up to 900,000 data points will be buffered when a connection to PIWebApi is lost.
```
Comment (TomKimsey) [inline README.md:38]:
```suggestion
The OSIsoft connector makes use of the Flexy's historical logging functionality to send historical data points from a Flexy to OSIsoft PIWebApi. The connector utilizes this functionality to buffer up to 900,000 data points when connection to PIWebApi is lost.
```

Comment (TomKimsey) [reply README.md:38]: i think saying "Due to" generally implies a negative connotation.

File: README.md [hunk 6]
```diff
@@ -281,6 +287,30 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib) to ensure all logging follows HMS standards for general connectors.
```
Comment (TomKimsey) [inline README.md:291]: > all logging follows HMS standards for general connectors

What does that mean?

File: README.md [hunk 7]
```diff
@@ -281,6 +287,30 @@ The Client ID and the Client Secret will be needed for the basic script you will
...
+
+To add a different logging level in your configuration file, specify the logging level under the AppConfig object.
```
Comment (TomKimsey) [inline README.md:293]: saying add a different implies there is a default. Should the default level (if one is not specified in the config) be documented here?

File: README.md [hunk 8]
```diff
@@ -281,6 +287,30 @@ The Client ID and the Client Secret will be needed for the basic script you will
...
+LoggingLevel is an optional field in the configuration file and is not required by the application to properly run.
+
+Below is an example of the configuration file set to log out general information.
+   ```
+   {
+   "ServerConfig":{
+      "IP":"192.168.1.1",
+      "WebID":"F1DS4knnwtuIDUC1RLy6XJGV4QREVTS1RPUC0wQ1MyUDFT",
+      "Credentials":"dsZtOnBfdsjklfh==",
```
Comment (TomKimsey) [inline README.md:302]:
```suggestion
      "Credentials":"cGFzc3dvcmQ=",
```

File: README.md (hunk 6 above)
Comment (oliver-walker-hms) [reply README.md:291]: just saying it uses the library sounded to short haha.
will change to:
The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib)

File: README.md (hunk 7 above)
Comment (oliver-walker-hms) [reply README.md:293]: It should not log at all if non is given, so I will add a issue to make 4 the default logging level later. Going to reword this

File: README.md [hunk 9]
```diff
@@ -281,6 +287,31 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib).
+
+To change the logging level in your configuration file, specify the logging level under the AppConfig object.
```
Comment (TomKimsey) [inline README.md:293]:
```suggestion
To change the logging level in the configuration file, specify the logging level under the AppConfig object.
```

File: README.md [hunk 10]
```diff
@@ -281,6 +287,31 @@ The Client ID and the Client Secret will be needed for the basic script you will
    
 You can now restart your Flexy to restart the Java application. Messages will be sent to your OCS endpoint.
 
+## JVM Logger
+The OSIsoft connector makes use of the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib).
```
Comment (TomKimsey) [inline README.md:291]:
```suggestion
The OSIsoft connector uses the [HMS Solution Center logging library](https://github.com/hms-networks/sc-flexy-logger-lib) for application logging to the Ewon Flexy's realtime logs.
```

Comment (TomKimsey) [reply README.md:291]: I think this might clarify what this is doing a little more to the average user.

File: README.md (hunk 9 above)
Comment (TomKimsey) [reply README.md:293]: 1st person

**Pull request merged**